rule/package-comments.go

```diff
@@ -34,6 +35,43 @@ func (*PackageCommentsRule) Apply(file *lint.File, _ lint.Arguments) []lint.Fail
 	return failures
 }
 
+var checkPackageCommentCache sync.Map
```

Comment:
> Is there any reason this can’t be defined as a field in PackageCommentsRule with a pointer field in lintPackageComments?

it sound more reasonable.